Extensions/ServiceNow/Src/vss-extension.json

```diff
@@ -4,8 +4,8 @@
   "name": "ServiceNow change request management",
```

Comment:
the id should be changed so it doesn't conflict with the private preview extension